src/ring_theory/power_series/basic.lean

```diff
@@ -341,6 +341,13 @@ begin
   simp only [X, coeff_mul_monomial, if_neg this]
 end
 
+lemma coeff_zero_X_mul (φ : mv_power_series σ R) (s : σ) :
```

Comment:
Is the left hand side in simp-normal-form? I think it is. If so, then this lemma (and the one above it) should be marked with `@[simp]`.
You can test this by just adding `@[simp]` and then writing `#lint` below the second lemma. If it doesn't complain, you're good.

Reply:
seems to work...

Reply:
`#lint` didn't who any errors, but apparently this caused a problem after all:
https://github.com/leanprover-community/mathlib/runs/1993170011
So I undid those changes.